src/cmd-kola

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+dn=$(dirname $0)
+. ${dn}/cmdlib.sh
+
+latest_qcow=$(get_latest_qemu)
+if [ -z "${latest_qcow}" ]; then
+    fatal "No latest build"
+fi
+
+# XXX: teach to kola
+distro=
+bn=$(basename "${latest_qcow}")
+if [[ ${bn} = fedora-coreos-* ]]; then
+    distro="-b fcos"
+elif [[ ${bn} = rhcos-* ]]; then
+    distro="-b rhcos --ignition-version v2"
+else
+    echo "WARNING: Failed to detect distro, use -b to specify"
+fi
+
+# let's print out the actual exec() call we do
+set -x
+
+# shellcheck disable=SC2086
+exec sudo kola $distro --output-dir tmp/kola --tapfile test.tap --qemu-image "${latest_qcow}" "$@"
```

Comment:
We don't have anything keying on the tapfile in this command, maybe drop it and let things specify it directly if they care about it?

Might be worthwhile to default to some extra parallel threads (via `--parallel <num>`).

Reply:
Thanks, dropped the `--tapfile` bit!

> Might be worthwhile to default to some extra parallel threads (via `--parallel <num>`).

I left this off for now. It's tricky to get right to match all possible environments. See e.g. `get_libvirt_smp_arg` in `virt-install`. Hmm, though I guess I could split that out into a separate Python script and here shell out to it?

Reply:
Probably fair to leave out `parallel` for now, it can be specified on the command-line and has the extra trickery around it as you mentioned.